packages/database/src/migrations/20201116024506-RemoveMapOverlayUNFPA-modifies-data.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  const servicesProvidedId = await db.runSql(`
+    SELECT id FROM map_overlay_group WHERE code = 'Services_provided'
+  `);
```

Comment:
Did the above code work for you locally?

```suggestion
  const { rows } = await db.runSql(`
    SELECT id FROM map_overlay_group WHERE code = 'Services_provided'
  `);
  const [{ id: servicesProvidedId }] = rows;
```

Similarly for the `down` migration

Reply:
@kael89 
It worked when I had the id hardcoded in postico (hah), and didn't throw an error when I ran the migration up/down locally in my terminal. Which probably misled me. I may need help with how to properly run multiple lines of code in positco's SQL query editor. Or perhaps async functionality is not possible? Might also need a 5 min demo of how to better test my own work. (One day if someone has time?)
Thank you! 

Have pulled the id out from the row by using your suggestion. Have another look.

Reply:
No worries @PShoe ! Happy to go through with you how you could test this feature locally and make sure that the functionality is correct